### PR TITLE
feat(suite): improve logging

### PR DESCRIPTION
**Pull request by tomasklim**

### CODE REVIEW

File: packages/suite/src/actions/suite/logActions.ts [hunk 1]
```diff
@@ -85,13 +91,14 @@ export const getLog =
 
 export const reportToSentry = (error: any) => (dispatch: Dispatch, getState: GetState) => {
     const { analytics, wallet, suite } = getState();
+
     Sentry.withScope(scope => {
         scope.setUser({ id: analytics.instanceId });
         scope.setContext('suiteState', {
             device: redactDevice(suite.device) ?? null,
-            discovery: wallet.discovery,
+            discovery: prettifyLog(wallet.discovery.map(d => redactDiscovery(d))),
```
Comment (matejkriz) [inline packages/suite/src/actions/suite/logActions.ts]: nit: no need for cryptic name
```suggestion
            discovery: prettifyLog(wallet.discovery.map(redactDiscovery)),
```

File: packages/suite/src/middlewares/suite/logMiddleware.ts [hunk 2]
```diff
@@ -46,25 +46,15 @@ const log =
...
-                    logActions.addCustom(action, {
-                        ...action.payload,
-                        firmwareRelease: undefined,
-                        release: undefined,
-                    }),
-                );
+                api.dispatch(logActions.addAction(action));
                 break;
             case SUITE.APP_CHANGED:
+                api.dispatch(logActions.addAction(action));
```
Comment (matejkriz) [inline packages/suite/src/middlewares/suite/logMiddleware.ts]: nit: those four cases could be handled at once, right?
```
case DEVICE.CONNECT:
case DEVICE.DISCONNECT:
case SUITE.APP_CHANGED:
case DISCOVERY.COMPLETE
```

**Comment by matejkriz** [comment]
@tomasklim please check my fixups

File: packages/suite/src/config/suite/sentry.ts [hunk 3]
```diff
@@ -3,6 +3,30 @@ import { BrowserOptions } from '@sentry/browser';
...
+    return event;
+};
+
+const beforeBreadcrumb: BrowserOptions['beforeBreadcrumb'] = breadcrumb => {
+    // filter analytics requests
+    if (
+        breadcrumb.category === 'fetch' &&
+        breadcrumb.data?.url?.contains('data.trezor.io/suite/log')
+    ) {
+        return null;
```
Comment (tomasklim) [inline packages/suite/src/config/suite/sentry.ts:28]: Breadcrumb is also full of fetches of images. Maybe we can filter them too?

Otherwise, its okay

Comment (matejkriz) [reply packages/suite/src/config/suite/sentry.ts:28]: good idea 👍 will do

**Review by matejkriz — APPROVED** [review]

**Pull request merged**